bootstrapform_jinja/templatetags/bootstrap.py

```diff
@@ -1,110 +1,148 @@
-from django import forms
+from django.forms import CheckboxInput, CheckboxSelectMultiple, \
+                         RadioSelect, FileInput
+from django.forms.boundfield import BoundField
 from django.template.loader import get_template
-from django_jinja import library
 from django.utils.safestring import mark_safe
+from django.utils.six import string_types
+from django_jinja import library
+
+from bootstrapform_jinja.config import BOOTSTRAP_COLUMN_COUNT
+
+
+class ClassSet(set):
+    """
+    Extension of a set that will concatenate elements in the set with
+    spaces when the set is converted to string.
+    """
+    def __str__(self):
+        return ' '.join((str(x) for x in self))
```

Comment:
If I remember right python 2.7 required extra parenthesis, but as that doesn't need to be supported those could be dropped for sure. Note to self: remember that map function exists...